src/Controls/samples/Controls.Sample.Sandbox/MauiProgram.cs

```diff
@@ -10,13 +17,47 @@ public static class MauiProgram
 		public static MauiApp CreateMauiApp() =>
 			MauiApp
 				.CreateBuilder()
+				.ConfigureMauiHandlers(handlers =>
+				{
+#if __ANDROID__ || __IOS__
+					handlers.AddMauiMapsControlsHandlers();
+#endif
+				})
+				.ConfigureLifecycleEvents(events =>
+				{
+#if __ANDROID__
+					// Log everything in this one
+					events.AddAndroid(android => android
+						.OnCreate((a, b) =>
+						{
+
+							Microsoft.Maui.Maps.Handlers.MapHandler.Bundle = b;
```

Comment:
Should this hook into `OnResume` ? 

When coming back from `Pause` only `OnResume` will get called.

Reply:
I copied this from Xamarin.Forms. I will test Pause is sending app to background?